src/Rector/Class_/MoveCurrentDateTimeDefaultInEntityToConstructorRector.php

```diff
@@ -155,6 +156,12 @@ private function refactorClass(Class_ $class, Property $property): void
             return;
         }
 
+        if ($defaultExpr instanceof ConstFetch
+            && $defaultExpr->name
+            && 'null' === $defaultExpr->name->toLowerString()) {
+            return;
+        }
+
```

Comment:
Maybe you can use somthing like:

```suggestion
        if ($this->valueResolver->isNull($defaultExpr)) {
            return;
        }

```